### PR TITLE
Update task.md

### PR DESCRIPTION
**Pull request by Ben-Rey**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/task.md [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
+about: Create a new task for the Run team to work on
 title: ''
 assignees: ''
-
 ---
-**Context**
-*Describe / explain why we should do this: motivations, context or other info. Keep it brief and simple - PM*
 
+**Task Overview**
+_Provide a concise description of the task, explaining what needs to be done, why it's important, and its impact on the project or user experience._
```
Comment (gregLibert) [inline .github/ISSUE_TEMPLATE/task.md:9]: everything is written in the same line when i view the markdown file. is it wanted ?

File: .github/ISSUE_TEMPLATE/task.md
Comment (gregLibert) [inline .github/ISSUE_TEMPLATE/task.md:1]: Are we using this template ?
This seems to be very dense for now don't you think ?

**Review by gregLibert — COMMENTED** [review]
:grey_question:

File: .github/ISSUE_TEMPLATE/task.md
Comment (Ben-Rey) [reply .github/ISSUE_TEMPLATE/task.md:1]: this is a question I asked myself, the problem for me is that it depends on the task. Maybe we can create multiple templates like simpleTask/complexTask

**Review by gregLibert — APPROVED** [review]

**Pull request merged**